core/src/main/kotlin/model/Documentable.kt

```diff
@@ -79,11 +79,15 @@ interface WithSupertypes {
     val supertypes: SourceSetDependent<List<TypeConstructorWithKind>>
 }
 
-interface Callable : WithVisibility, WithType, WithAbstraction, WithExpectActual {
+interface WithIsExpectActual {
```

Comment:
I like changing `WithExpectActual` to `WithSources`, but I don't see a reason why `Callable is not extending `WithIsExpectedActual`.

Reply:
My fault, actually I extended `DFunction` and `DProperty` with `WithIsExpectActual`, didn't realise I can aggregate it via `Callable` interface. Will change